NOnion/Utility/ObservableUtils.fs

```diff
@@ -0,0 +1,13 @@
+﻿namespace NOnion.Utility
+
+open System
+
+module ObservableUtils =
```

Comment:
this should maybe be removed in the "Restructure" commit?

Reply:
Removed.